charts/clamav/values.yaml

```
@@ -0,0 +1,126 @@
+    cvdCron: "*/10 * * * *"
+    # clamavmirror.cvdUpdate.MirrorCron -- Frequency to rsync changes to mirror - defaults to every hour   
+    mirrorCron: "0 * * * *"
+  metricsPort: 9090
+  volume:
+    # clamavmirror.volume.storageClass -- The storage class of the volume used by clamav-mirror
+    storageClass: gp2-encrypted
+    # clamavmirror.volume.size -- The size class of the volume used by clamav-mirror in GB
+    size: 10
+
+clamavHTTP:
+  # clamavHTTP.image -- The clamav-http docker image
+  image: quay.io/ukhomeofficedigital/acp-clamav-http
+  # clamavHTTP.version -- The clamav-http docker image version - defaults to .Chart.appVersion
+  version: ""
+  resources:
+    # clamavHTTP.resources -- The resource requests and limits for the clamav-http service
+    requests:
+      cpu: 200m
+      memory: 2000M
+      ephemeral-storage: "500M"
+    limits:
+      cpu: 1000m
+      memory: 2000M
+      ephemeral-storage: "1000M"
+
+clamav:
+  # clamav.image -- The clamav docker image
+  image: quay.io/ukhomeofficedigital/acp-clamav 
+  # clamav.version -- The clamav docker image version - defaults to .Chart.appVersion
+  version: ""
+  freshclam:
+    # clamav.freshclam.mirrors -- A list of clamav mirrors to be used by the clamav service
+    mirrors:
+      - database.clamav.net
+  certificate:
+    # clamav.certificate.create -- Should a certificate be created for the clamav service
+    create: true
+    # clamav.certificate.secretName -- Override the default certificate secret name
+    secretName: ""
+    # clamav.certificate.issuer -- The issuer to use when creating a certificate
+    issuer:
+      name: platform-ca
+      kind: ClusterIssuer
+  scaling:
+    # clamav.scaling.minReplicas -- The minumum number of clamav replicas
+    minReplicas: 2
+    # clamav.scaling.maxReplicas -- The maximum number of clamav replicas
+    maxReplicas: 20
+    # clamav.scaling.cpuTarget -- The target cpu usage percentage for clamav
+    cpuTarget: 30
+  limits:
+    # clamav.limits.fileSize -- The largest file size scanable by clamav, in MB
+    fileSize: 200
+    # clamav.limits.scanSize -- The largest scan size permitted in clamav, in MB
+    scanSize: 1000
+    # clamav.limits.connectionQueueLength -- Maximum length the queue of pending connections may grow to
+    connectionQueueLength: 200
+    # clamav.limits.maxThreads --Maximum number of threads running at the same time.
+    maxThreads: 10
+    # clamav.sendBufTimeout -- This option specifies how long to wait (in milliseconds) if the send buffer is full, keep low to avoid clamd hanging
+    sendBufTimeout: 500
+
+
+  resources:
+    # clamav.resources -- The resource requests and limits for the clamav service
+    requests:
+      cpu: 200m
+      memory: 2000M
+      ephemeral-storage: "500M"
+    limits:
+      cpu: 1000m
+      memory: 4000M
+      ephemeral-storage: "1000M"
+nginxProxy:
```

```suggestion

nginxProxy:
```

Confused me a little without the space.